source/geod24/LocalRest.d

```diff
@@ -701,25 +697,32 @@ public final class RemoteAPI (API) : API
                         },
                         (Response res) {
                             if (!isSleeping())
+                            {
```

Comment:
Please keep the formatting changes, such as adding braces around a single statement, in their own commit